vulture_os/darwin/migrations/0010_auto_20201118_1301.py

```diff
@@ -0,0 +1,165 @@
+# Generated by Django 2.1.3 on 2020-11-18 13:01
+
+import darwin.policy.models
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+import djongo.models.fields
+from toolkit.mongodb.mongo_base import MongoBase
+
+
+def set_enrichment_tags(apps, schema_editor):
+    mongo = MongoBase()
+    if not mongo.connect(node=mongo.get_local_node()):
+        raise ConnectionError("Cannot connect to mongodb")
```

Comment:
Please do not raise, otherwise you will crach all the update routine.
Also, please connect to Master, not local node, to **write** into cluster.

Reply:
Migration script has been updated to try and connect to primary node.
Update scripts have been moved to 'v1.1.0'
I didn't merge them though, as one is not related to Darwin policies, and I find it arguably easier to debug and redo a single script (representing a step) rather than all scripts (and all scripts can be run independently)